tests/keep/broken-json-note.json

```diff
@@ -0,0 +1,10 @@
+{
+    "color": "DEFAULT",
+    "isTrashed": false,
+    "isPinned": false,
+    "isArchived": false,
+    "textContent": "This is just a plain text note.     This sentence has more than one space before it.\n\n\nThis sentence has more than one new line before it.\n\n   This sentence has more than one new line and spaces before it.",
+    "title": "_Text note"
```

Comment:
I don't think we need to test whether the JSON parser can detect broken files. I think the actual exported zip file is good enough.

Reply:
You wouldn't wanna know if it can break with unexpected input?

Reply:
I don't think we need to test the browser implementation of JSON.parse

Reply:
Not for that, but to Test that the keep format-importer code is correctly try/catching and reporting.
(For the broken file)

And the other file tests that it gracefully handles and reports a valid json that isn't an valid keep json.

In both cases it should show that it fails in the output log.

Reply:
Oh ok, fair enough.